go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/mitchellh/copystructure v1.0.0 // indirect
 	github.com/mitchellh/reflectwalk v1.0.0 // indirect
 	github.com/shopspring/decimal v1.2.0 // indirect
+	github.com/shurcooL/graphql v0.0.0-20230714182844-3e04114ae69a // indirect
```

Comment:
looks like the generator still uses the outdated library - we should update that and then it won't need this in go.mod

Reply:
Changed to https://github.com/hasura/go-graphql-client